Team-Kangaroo/space-barter-backend
# [MOD] Fixed Issue With Twilio Auth

#### What does this PR do?
Fixes issue with Twilio Auth.

#### How should this be manually tested?
Use the phone verification as normal. It should work now.
`POST /verify?phoneNumber=081XXXXXXXX`
